packages/app/lib/src/features/home/presentation/home/post.dart

```diff
@@ -58,6 +57,59 @@ class _PosterInfo extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
+    // Have post info show how long ago in bar
+    // Show actual date and time of post if you click on it
+    final timeSincePost = post.timestamp.difference(DateTime.now()) +
+        post.timestamp.timeZoneOffset; // Duration values are negative
+    int timeValue;
+    String timePostValue;
+    if (timeSincePost.inDays < -364) {
+      //TODOwrite more efficient code with variables
+      timeValue = (timeSincePost.inDays / -364).round();
+      switch (timeValue) {
+        case 1:
+          timePostValue = '$timeValue year ago';
+        default:
+          timePostValue = '$timeValue years ago';
+      }
+    } else if (timeSincePost.inDays <= -1) {
+      timeValue = timeSincePost.inDays.abs();
```

Comment:
abs is absolute value, right? When would we get a negative value for this?

Reply:
timeSincePost is a negative since it's a duration and we're comparing a past value to a current.